common/src/OpenSpaceNet.cpp

```diff
@@ -625,16 +629,9 @@ FileFeatureSink::Ptr OpenSpaceNet::initFeatureSink()
 
 void OpenSpaceNet::startProgressDisplay()
 {
-    if(!pd_ || args_.quiet) {
-        return;
+    if(pd_ && !args_.quiet) {
```

Comment:
since this is all that remains, consider inlining the remaining 3 lines

Reply:
No, I'll keep it here. This way there's room for logic. There's enough inlined crap in `OpenSpaceNet::process` already.